test/runtests.jl

```diff
@@ -1,5 +1,5 @@
-include("TestComponents.jl")
-using .TestComponents
+#include("TestComponents.jl")
+#using .TestComponents
```

Comment:
If these are no longer needed (looks like because dcc and html are included in this repo now?) can we delete these lines and the `TestComponents.jl`file?

Similar question about `dev.jl` - you made changes to that file but it's commented out below, is it used?

Reply:
>can we delete these lines and the TestComponents.jlfile?
done in 87533d9